src/core/tests/type_prop/extractimagepatches.cpp

```diff
@@ -164,3 +164,15 @@ TEST(type_prop, extractimagepatches_large_stride_same_padding) {
     EXPECT_EQ(extractimagepatches->get_output_element_type(0), element::i32);
     EXPECT_EQ(extractimagepatches->get_output_shape(0), (Shape{64, 27, 1, 1}));
 }
+
+TEST(type_prop, extractimagepatches_dyn) {
```

Comment:
@v-Golubev also add tests for `ExtractImagePatches` with `PartialShape::dynamic()`